src/components/Message/MessageSimple/MessageTextContainer.tsx

```diff
@@ -0,0 +1,193 @@
+import React, { useContext } from 'react';
+
+import { capitalize } from './utils/capitalize';
+import { renderText } from './utils/renderText';
+
+import { styled, ThemeContext } from '../../../styles/styledComponents';
+
+import type { MessageSimpleProps } from './MessageSimple';
+import type {
+  Alignment,
+  GroupType,
+} from '../../../contexts/messagesContext/MessagesContext';
+import type { Message as MessageType } from '../../../components/MessageList/utils/insertDates';
+import type { MarkdownStyle, Theme } from '../../../styles/themeConstants';
+import type {
+  DefaultAttachmentType,
+  DefaultChannelType,
+  DefaultCommandType,
+  DefaultEventType,
+  DefaultMessageType,
+  DefaultReactionType,
+  DefaultUserType,
+} from '../../../types/types';
+
+const TextContainer = styled.View<{
+  alignment: string;
+  groupStyle: string;
```

Comment:
GroupType?

Reply:
kinda confusing but no, the component takes `groupStyles` which is of GroupType, but `groupStyle` is slightly different and can be other values like "leftBottom" or "rightMiddle"